plugins/test/generator.cpp

```diff
@@ -9,38 +9,50 @@
 //
 
 #include <mrdox/Plugin.hpp>
+#include <mrdox/Generator.hpp>
+#include <mrdox/Generators.hpp>
 
-struct myGen final : clang::mrdox::Generator
+struct myGen final : public clang::mrdox::Generator
 {
-  std::string_view
-  id() const noexcept override
-  {
-    return "test";
-  }
-
-  std::string_view
-  displayName() const noexcept override
-  {
-    return "test generator";
-  }
-
-  std::string_view
-  fileExtension() const noexcept override
-  {
-    return "test";
-  }
-
-  clang::mrdox::Error
-  buildOne(
-      std::ostream& os,
-      clang::mrdox::Corpus const& corpus) const override
-  {
-    os << "It works!";
-    return {};
-  }
+    std::string_view
+    id() const noexcept override {
+        return "test";
+    }
+
+    std::string_view
+    displayName() const noexcept override {
+        return "test generator";
+    }
+
+    std::string_view
+    fileExtension() const noexcept override {
+        return "test";
+    }
+
+    clang::mrdox::Error
+    buildOne(
+            std::ostream &os,
+            clang::mrdox::Corpus const &corpus) const override
+            {
+        os << "It works!";
```

Comment:
cool heh

Reply:
That'll cause an error of undefined symbols on startup.